### DISCUSSION TITLE
Used Skraper, but no images on main page or grid

### DISCUSSION THREAD

**Discussion by DeltaGemini** [PlayingKarrde/gameOS#177] [Q&A]
I just used Skraper on Windows for my Retropie and videos now play when I have a game highlighted. I can also see the image mix when showing full game details.

However, there are no images in the game tiles on either the grid or the home screen, only game titles. Am I missing a setting in Skraper?

Many thanks!

**Comment by PlayingKarrde** [comment]
Are you scraping screenshots? The tiles in gameOS use a combination of clear logo and screenshot.

**Reply by DeltaGemini** [reply]
Thanks! I'll take a look 🙂

**Reply by brunomueller93** [reply]
Can you post a quick guide? I've been trying to scrape with Skyscraper but somehow nothing shows up. Apart from the default settings of Skyscraper that I think also scrape screenshots, I've also scraped videos. 

In Skyscraper you can specifically tell it to scrape for Pegasus and a metadata file is generated buuuut nothing shows up in gameOS. It works with the default Pegasus theme but not in gameOS.
